api/pkg/dataflow/service.go

```diff
@@ -420,6 +423,11 @@ func (s *APIService) GetJob(request *restful.Request, response *restful.Response
 	ctx := common.InitCtxWithAuthInfo(request)
 	res, err := s.dataflowClient.GetJob(ctx, &dataflow.GetJobRequest{Id: id})
 	if err != nil {
+		if strings.Contains(err.Error(), "job not exist") || strings.Contains(err.Error(), "invalid input to ObjectIdHex") {
```

Comment:
Is it possible to put these strings as var and refer them here? These seems to be generic which can be referred elsewhere too